tmk_core/protocol/ps2_interrupt.c

```diff
@@ -232,40 +261,62 @@ static uint8_t     pbuf[PBUF_SIZE];
 static uint8_t     pbuf_head = 0;
 static uint8_t     pbuf_tail = 0;
 static inline void pbuf_enqueue(uint8_t data) {
+#if defined(__AVR__)
     uint8_t sreg = SREG;
     cli();
+#endif
+
```

Comment:
I believe the Chibios analog for this would be `chSysLockFromISR` or `osalSysLockFromISR`.

Reply:
changed with 33f99188f62d5aabea04dc9ddbf3188c1cf49d4d